MemberExpectations.md

```diff
@@ -0,0 +1,63 @@
+
+Members are encouraged to have individual opinions and are encouraged
+to express those opinions during discussions within the organization. While
+discourse is encouraged internally, members also have the obligation outside
+of the organization (for example in social media) to be respectful of the
+decisions made within the projects's documented processes.  A member who does
+not support a team decision or current team value may express the member's
+opposition within the team in an appropriate manner, but must be extremely
+careful when expressing the opinion externally.  External communication must
+not be done with the intent of undermining a decision, project values, or
+actions of the team. If another team member expresses that they believe external
+communications appear to have the intent to undermine the committee
+it is the responsibility of the individual expressing their opinions to rectify
+the situation immediately. If a member feels that there is no recourse aside
+from going public regarding an extremely contentious issue then they should
+resign from the team before doing so.
+
+In addition to being willing to act as a team player, members of our
+leadership groups must conduct themselves in a professional and
+respectful manner. Some general
+guidelines include:
+
+- Serve as ambassadors of the vision, mission and operating
+  principles of the Node.js project.
+- Remediate quickly when you realize you made a mistake. Leaders are human,
+  and they will make mistakes - however they should act swiftly to
+  acknowledge mistakes and correct them. Most often this can mitigate
+  any damage done.
+- Aim to remediate first and then discuss.  If other members of the
+  team express concerns about actions, acknowledge their concerns by
+  stopping the actions in question and then discuss within the team
+  to come to a common agreement.
+- Treat all community members with respect, consideration, and highest
+  standards of ethical conduct.  
+- Valuing a diversity of views and opinions. Strive to avoid preferential
+  treatment, and hold everyone (including ourselves) accountable to the same
+  set of standards.  Everyone gets to speak up. 
+- Deal with issues directly with the person in question. Resist complaining
+  about others in the project in a public sphere.
+- Built trust by keeping your promises.
+- Be the model of accountability and leadership. Provide the example of
+  ownership and stewardship that everyone can follow to success.
+- Commit to ongoing development and learning best practices for governing.
+- Critique ideas rather than individuals, discussing any concerns in person
+  whenever possible, and taking responsibility for our statements by
+  speaking as much as possible
+  in the first person (.I. statements) rather than in the third person.
+- Collectively and as an individual serve as a role model to demonstrate
+  the highest standards of ethical conduct.
```

Comment:
> Collectively and as an individual serve as a role model to demonstrate the highest standards of ethical conduct.

is a double of 

> Treat all community members with respect, consideration, and highest standards of ethical conduct.

I prefer putting the focus of treating everyone with high respect is more important than (aspiring) to be role models. I do not think one can decide to be "a role model", it's the action of others that decide that you are.